BUILDING.md

```diff
@@ -206,8 +205,7 @@ in the current continuous integration environment. The participation of people
 dedicated and determined to improve Android building, testing, and support is
 encouraged.
 
-Be sure you have downloaded and extracted [Android NDK]
-(https://developer.android.com/tools/sdk/ndk/index.html)
+Be sure you have downloaded and extracted [Android NDK](https://developer.android.com/tools/sdk/ndk/index.html)
```

Comment:
Could you do it like:

```
Download and extract the
[Android NDK](https://developer.android.com/tools/sdk/ndk/index.html),
then run:
```

? That way you keep it <80 chars and the link should still work.